rcl/src/rcl/context.c

```diff
@@ -129,8 +130,11 @@ __cleanup_context(rcl_context_t * context)
 
     // finalize init options if valid
     if (NULL != context->impl->init_options.impl) {
-      rcl_ret_t ret = rcl_init_options_fini(&(context->impl->init_options));
-      if (RCL_RET_OK != ret) {
+      rcl_ret_t inner_ret = rcl_init_options_fini(&(context->impl->init_options));
```

Comment:
Instead of `inner_ret` maybe `init_options_fini_ret`

Reply:
Sounds good. See https://github.com/ros2/rcl/pull/763/commits/6f3cab1371766922ad1422d03f1259766c11d417.